apps/whale-api/src/module.api/consortium.controller.spec.ts

```diff
@@ -3,125 +3,126 @@ import { TestingGroup } from '@defichain/jellyfish-testing'
 import { createTestingApp, stopTestingApp } from '../e2e.module'
 import { NestFastifyApplication } from '@nestjs/platform-fastify'
 import { StartFlags } from '@defichain/testcontainers'
+import { NotFoundException } from '@nestjs/common'
 
-describe('getAssetBreakdown', () => {
-  const tGroup = TestingGroup.create(2)
-  const alice = tGroup.get(0)
-  const bob = tGroup.get(1)
-  const symbolBTC = 'BTC'
-  const symbolETH = 'ETH'
-  let accountAlice: string, accountBob: string
-  let idBTC: string
-  let idETH: string
-  let app: NestFastifyApplication
-  let controller: ConsortiumController
-  const startFlags: StartFlags[] = [{ name: 'regtest-minttoken-simulate-mainnet', value: 1 }]
-
-  beforeEach(async () => {
-    await tGroup.start({ startFlags })
-    await alice.container.waitForWalletCoinbaseMaturity()
-
-    app = await createTestingApp(alice.container)
-    controller = app.get(ConsortiumController)
+const tGroup = TestingGroup.create(2)
+const alice = tGroup.get(0)
+const bob = tGroup.get(1)
+const symbolBTC = 'BTC'
+const symbolETH = 'ETH'
+let accountAlice: string, accountBob: string
+let idBTC: string
+let idETH: string
+let app: NestFastifyApplication
+let controller: ConsortiumController
+const startFlags: StartFlags[] = [{ name: 'regtest-minttoken-simulate-mainnet', value: 1 }]
+
+beforeEach(async () => {
+  await tGroup.start({ startFlags })
+  await alice.container.waitForWalletCoinbaseMaturity()
+
+  app = await createTestingApp(alice.container)
+  controller = app.get(ConsortiumController)
+})
+
+afterEach(async () => {
+  await stopTestingApp(tGroup, app)
+})
+
+async function setGovAttr (attributes: object): Promise<void> {
+  const hash = await alice.rpc.masternode.setGov({ ATTRIBUTES: attributes })
+  expect(hash).toBeTruthy()
+  await alice.generate(1)
+  await tGroup.waitForSync()
+}
+
+async function setMemberInfo (tokenId: string, memberInfo: Array<{ id: string, name: string, backingId: string, ownerAddress: string, mintLimit: string, dailyMintLimit: string }>): Promise<void> {
+  const members: any = {}
```

Comment:
all  good, just one nitpick , this is better? 
```suggestion
const members: { [key: string]: { [key: string]: string } } = {}
```